ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,33 +191,64 @@ const POPULATIONS = [
     queryParams: { [GENOME_VERSION_37]: 'dataset=gnomad_sv_r2_1' },
     helpMessage: GNOMAD_SV_CRITERIA_MESSAGE,
   },
+  {
+    field: 'callset_heteroplasmy',
+    fieldTitle: 'This Callset',
+    acDisplay: 'AC',
+    isHet: true,
+    precision: 3,
+  },
   {
     field: 'gnomad_mito',
     fieldTitle: 'gnomAD mito',
     precision: 3,
-    urls: { [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
-    queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+    ...GNOMAD_URL_INFO,
+  },
+  {
+    field: 'gnomad_mito_heteroplasmy',
+    fieldTitle: 'gnomAD mito',
+    isHet: true,
+    precision: 3,
+    ...GNOMAD_URL_INFO,
   },
   {
     field: 'helix',
     fieldTitle: 'Helix mito',
     precision: 3,
   },
+  {
+    field: 'helix_heteroplasmy',
+    fieldTitle: 'Helix mito',
+    isHet: true,
+    precision: 3,
+  },
 ]
 
 const Frequencies = React.memo(({ variant }) => {
   const { populations = {} } = variant
-  const freqContent = <div>{POPULATIONS.map(pop => <FreqSummary key={pop.field} variant={variant} {...pop} />)}</div>
+  const isMito = POPULATIONS.some(pop => pop.isHet &&
+      (populations[pop.field]?.af !== null && populations[pop.field]?.af !== undefined))
+  const freqContent = (
+    <div>
+      {POPULATIONS.map(pop => <FreqSummary key={pop.field} variant={variant} isMito={isMito} {...pop} />)}
+    </div>
+  )
 
-  const hasAcPops = POPULATIONS.filter(pop => populations[pop.field] && populations[pop.field].ac)
+  const hasAcPops = POPULATIONS.filter(pop => !pop.isHet && populations[pop.field]?.ac)
```

Comment:
the `isHet` specific logic here makes this really hard to read and maintain. I think it would be better if you made the logic more generic so it could be extended easily and we wouldn't have mito-specific logic hard coded in what it supposed to be a generic component. Add something like `detailSection: 'Hetroplasmy'` to the population configs, and then here you would add a check for `!pop.detailSection`. Then below you could do something to get a list of relevant detail sections as follows and then render them:
```
const detialSections = POPULATIONS.reduce((acc, pop) => {
  if (pop.detailSection && populations[pop.field]) {
    if (!acc[pop.detailSection]) {
      acc[pop.detailSection] = []
    }
    acc[pop.detailSection].push(populations[pop.field])
  }
  return acc
}, {})
```

Reply:
If I understand correctly, I will use the name `sectionHeader` for the `detailSection` you mentioned.